### DISCUSSION TITLE
v8 'DateUtils' is not exported from 'react-day-picker'

### DISCUSSION THREAD

**Discussion by vlad-ciorba** [gpbl/react-day-picker#1489] [Support] [answered]
Hi,
I got this error, `'DateUtils' is not exported from 'react-day-picker'`, after updating from v7 to v8 and there is no documentation regarding renaming or removing of DateUtils from react-day-picker.
Any idea of what could be the issue?
Thank you.

**Comment by kevindcode** [comment]
Yeah I just bumped into this also!

Nothing in the **[migration guide](https://react-day-picker.js.org/guides/upgrading)** either 😢

**Comment by McSam27** [comment]
I also found a few things were missing from the migration guide... need to find that list again.

**Comment by gpbl** [comment, marked as answer]
`DateUtils` is not there anymore as it has been replaced by date-fns. @stopr29  added `addToRange` to the exported utilities in #1495, which should be released next.

I've added some docs about `DateUtils` in 9bfaf235fe3c5d77f48761c0c5f0c7353f184159.

Thanks for the feedback!
